### ISSUE TITLE
Sad, some Windows versions kernelbase.dll can't find rdi gad

### ISSUE THREAD

**Issue by D1sAbl4** [NtDallas/Fenrir#1]
**Find Rdi gad**
![image](https://github.com/user-attachments/assets/6a3849d9-0d06-4813-bd02-73eb91f9ae2f)

- `win 22H2, 10.0.19045.5131`
![image](https://github.com/user-attachments/assets/e3eec7d8-ee34-4b7c-8dde-410485dcda7d)

- `winserver 2019, 10.0.17763.6189`
![image](https://github.com/user-attachments/assets/b6035aa2-ab2d-4814-b79d-c65687ebfc63)

**Comment by NtDallas** [comment]
Yes, I mentioned in the README that I tested it on Windows 11 24H2, not on other versions of Windows

I noticed that on Windows Server, no gadget is present. When I have time, I’ll check how to use other gadgets in case "jmp RDI" is not available.
